great_expectations/experimental/metric_repository/metrics.py

```diff
@@ -18,12 +33,8 @@ class Config:
 
 
 class MetricException(MetricRepositoryBaseModel):
-    exception_type: Optional[str] = Field(
-        description="Exception type if an exception is thrown", default=None
-    )
-    exception_message: Optional[str] = Field(
-        description="Exception message if an exception is thrown", default=None
-    )
+    type: str = Field(description="Exception type if an exception is thrown")
+    message: str = Field(description="Exception message if an exception is thrown")
```

Comment:
I forgot to rename the fields to `type` and `message` instead of `exception_type` and `exception_message` in https://github.com/great-expectations/great_expectations/pull/8648 to match the backend.